v2/telemetry.js

```diff
@@ -312,6 +312,10 @@
       });
     };
 
+    Evolution.prototype.metric_count = function () {
```

Comment:
telemetry.js uses `camelCase` for its API names.

Speaking of which, if you're adding to the API you need to add to the documentation. telemetry.js is used in a few places by a few teams and individuals. Keeping the documentation up-to-date is of utmost importance.